README.md

```diff
@@ -23,236 +63,324 @@ cd lndg
 # Customize the docker-compose.yaml file
 nano docker-compose.yaml
 ```
-**Replace the contents of `docker-compose.yaml` with your desired volume paths and settings. An example configuration is shown below.**
+
+Replace the contents of `docker-compose.yaml` with your desired volume paths and settings. An example configuration is shown below:
+
 ```yaml
 services:
   lndg:
     build: .
     volumes:
+      # Adjust paths according to your setup
       - /home/<user>/.lnd:/root/.lnd:ro
       - /home/<user>/<path-to>/lndg/data:/app/data:rw
     command:
       - sh
       - -c
+      # Customize network ('mainnet' or 'testnet') and RPC address/port if needed
       - python initialize.py -net 'mainnet' -rpc '127.0.0.1:10009' -wn && python controller.py runserver 0.0.0.0:8889 > /var/log/lndg-controller.log 2>&1
+    # Use host network mode for simplicity, adjust if needed
     network_mode: "host"
 ```
-### Build and Deploy
+
+**2. Build and Deploy:**
+
 ```bash
-# Deploy the Docker image
+# Build and deploy the Docker container in detached mode
 docker-compose up -d
 
-# Retrieve the admin password for login
-nano data/lndg-admin.txt
+# Retrieve the admin password for the initial login
+cat data/lndg-admin.txt
 ```
-- **This example configuration will host LNDg at http://0.0.0.0:8889. Use the machine IP to reach the LNDg instance.**  
-- **Log in to LNDg using the provided password and the username `lndg-admin`.**
 
-### Updating
+- This example configuration will host LNDg at `http://<your-machine-ip>:8889`.
+- Log in to LNDg using the username `lndg-admin` and the password retrieved from `data/lndg-admin.txt`.
+
+### Manual Installation
+
+This method provides a hands-on approach to setting up LNDg.
+
+**1. Install LNDg:**
+
 ```bash
+# Clone the repository
+git clone https://github.com/cryptosharks131/lndg.git
+
+# Change directory
+cd lndg
+
+# Ensure Python virtualenv is installed (example for Debian/Ubuntu)
+sudo apt update && sudo apt install -y virtualenv
+
+# Set up a Python 3 virtual environment
+virtualenv -p python3 .venv
+
+# Activate the virtual environment (syntax might vary based on shell)
```

Comment:
We shouldn't need to activate here as below we use a full path to reference the virtual env python and pip binaries.